rfc/rfc023-x509credential.md

```diff
@@ -0,0 +1,453 @@
+that the certificate is valid and that the information in the certificate is correct. The UZI certificates are issued
+to:
+
+* Individuals that work in healthcare, such as doctors, nurses, etc. They hold this certificate on a UZI card.
+* Organisations that work in healthcare, such as hospitals, pharmacies, etc. They hold this certificate as server
+  certificates.
+
+#### UZI certificate structure for organisations
+
+The UZI certificate is used to identify the holder of the certificate. The UZI certificate contains information about
+the holder of the certificate. This information is used to identify the holder of the certificate. The UZI certificate
+contains the following information (of intrest):
+
+* The `subject.CN` The full FQN.
+* The `subject.O` the name of the holder of the certificate.
+* The `subject.serialNumber ` The URI number
+* The `subject.C` The subject country
+* The `subject.ST` The subject state
+* The `subject.L` The subject locality (city)
+* The `subject.commonName` the full FQN.
+* The `san.dNSName` the DNS name of the holder of the certificate.
+* The `san.otherName` a string containing `<OID CA>-<versie-nr>-<UZI-nr>-<pastype>-<Abonnee-nr>-<rol>-<AGB-code>`,
+  where:
+  * `<OID CA>` is the OID of the CA that issued the certificate, `2.16.528.1.1007.99.2110` for CIBG.
+  * `<versie-nr>` is the version number of the certificate.
+  * `<UZI-nr>` is the UZI number of the holder of the certificate, same as `subject.serialNumber`.
+  * `<pastype>` is the type of the holder of the certificate, always `S`.
+  * `<Abonnee-nr>` is the subscriber URA of the holder of the certificate.
+  * `<rol>` is the role of the holder of the certificate, always "0.00"
+  * `<AGB-code>` is the AGB code of the holder of the certificate.
+
+## Mapping UZI certificate to X509Credential
+
+The mapping of certificates to x509 is depending
+
+### The ROOT G3
+
+The `did:x509` specification dictates that the fingerprint of the Root CA is part of the did:x509. For mapping an UZI
+certificate to an X509Credential the ROOT CA MUST match one of the certificates in the UZI ROOT CA register hierarchy.
+For G3 this is:
+
+```asciidoc
+        ┌────────────────────────────────────┐        
+        │ Staat der Nederlanden Root CA - G3 │        
+        └────────────────┬───────────────────┘        
+                         │                            
+┌────────────────────────▼───────────────────────────┐
+│ Staat der Nederlanden Organisatie Services CA - G3 │
+└────────────────────────┬───────────────────────────┘
+                         │                            
+    ┌────────────────────▼───────────────────────┐    
+    │ UZI-register Medewerker niet op naam CA G3 │    
+    └────────────────────────────────────────────┘    
+```
+
+### Field mapping of the UZI credential
+
+The following fields are commonly used for mapping UZI certificates to X509Credentials
```

Comment:
We are specifying it here how it should be used, not how it is used. So rephrase to something like: The following fields MUST be included in the mapping.